pandas/src/hashtable_class_helper.pxi.in

```diff
@@ -237,6 +238,12 @@ cdef class {{name}}HashTable(HashTable):
         k = kh_get_{{dtype}}(self.table, key)
         return k != self.table.n_buckets
 
+    def sizeof(self, deep=False):
+        """ return the size of my table in bytes """
+        return self.table.n_buckets * (sizeof(self.table.keys) +
```

Comment:
`sizeof` will return a pointer size here - I think you instead need to do something like this? (not sure the kh aliases are exposed)

```
self.table.n_buckets * (sizeof(khkey_t)  + #keys
                        sizeof({{dtype}}_t)  + #vals
                        sizeof(khint32_t))  #flags
```
                           
https://github.com/pandas-dev/pandas/blob/be328521b3a5525175b08cb740bb970ccf75727e/pandas/src/klib/khash.h#L189